e2e/scenario_win_test.go

```diff
@@ -70,8 +87,60 @@ func Test_Windows23H2Gen2(t *testing.T) {
 			VHD:             config.VHDWindows23H2Gen2,
 			VMConfigMutator: func(vmss *armcompute.VirtualMachineScaleSet) {},
 			BootstrapConfigMutator: func(configuration *datamodel.NodeBootstrappingConfiguration) {
-
+			},
+			Validator: func(ctx context.Context, s *Scenario) {
+				ValidateFileHasContentWindows(ctx, s, "/k/kubeletstart.ps1", "--container-runtime=remote")
+				//ValidateProcessHasCliWindows(ctx, s, "kubelet.exe", []string{"--address=0.0.0.0", "--anonymous-auth=false"})
 			},
 		},
 	})
 }
+
+func makeExecutablePowershellCommand(steps []string) string {
```

Comment:
can we move the validation-related code over to validation.go?

Reply:
Yes! Will do this.